lib/src/generator/templates.dart

```diff
@@ -2,8 +2,6 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// Annotations only appear on other pages, so there is no template data here.
-@Renderer(#renderAnnotation, Context<Annotation>())
```

Comment:
I don't know why it would have been required before. It would only be required if the generated method, `renderAnnotation` is used somewhere, like in templates.dart or template_data.dart, or in backend_generator.dart.

The class might have been invisible to mustache while some code was WIP, but is now found in the `_visibleTypes` set.